examples/oraclelinux.yaml

```diff
@@ -0,0 +1,17 @@
+# Licence: https://www.oracle.com/downloads/licenses/oracle-linux-license.html
```

Comment:
Please clarify that this license applies to the image, not to the YAML itself, which is licensed under the terms of Apache License 2.0

Reply:
Clarified: https://github.com/lima-vm/lima/pull/700/files#diff-ec074ef274eaabe9a71610791c8ed66c88b59ba7bb648390f2c34426d8130a07R1